pages/linux/dd.md

```diff
@@ -17,3 +17,7 @@
 - Benchmark the write performance of a disk:
 
 `dd if=/dev/zero of={{file_1GB}} bs=1024 count=1000000`
+
+- Check dd progress (Run this command in another shell then check dd output):
```

Comment:
I think `Check progress of an ongoing dd operation (Run this command from another shell):` sounds better.

Reply:
Thanks, I've updated the comment